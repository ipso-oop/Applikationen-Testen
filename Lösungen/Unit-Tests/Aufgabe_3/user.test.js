const { getUser } = require('./user');


test('User-Daten = Snapshot', () => {
  expect(getUser()).toMatchSnapshot();
});
