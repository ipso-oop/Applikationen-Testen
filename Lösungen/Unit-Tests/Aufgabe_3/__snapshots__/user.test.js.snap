// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User-Daten = Snapshot 1`] = `
{
  "age": 31,
  "name": "Alice",
}
`;

exports[`User-Daten entsprechen Snapshot 1`] = `
{
  "age": 30,
  "name": "Alice",
}
`;
